rotkehlchen/chain/ethereum/modules/uniswap/v2/decoder.py

```diff
@@ -1,6 +1,7 @@
 from typing import Callable, Optional
 
 from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
```

Comment:
> Now, you might begin to wonder, "why doesn't a bugfix contain a test?"... A test actually does exist however, the user doesn't want to be doxxed and multiple attempts at finding transactions that match this edge case have been futile.
> 
> As a result of this, whoever will be reviewing this PR I'll share a diff that contains the test. Sorry for any incoveniences
> 
> TLDR: Closes #5722

Please use the data you have and add the test in the codebase without using VCR and hard-coding the transaction and its log events, but changing hashes and addresses.

I won't even look at a PR for a bugfix without a test.